README.md

```diff
@@ -94,6 +95,19 @@ An additional dimension that is not `spatial`, but may be `temporal` if the data
 
 An Additional Dimension Object MUST specify an `extent` or a set of `values`. It MAY specify both.
 
+### Variable Object
+
+A *Variable Object* defines a multi-dimensional array. The array has dimensions, which are described by [Dimension Objects](#dimension-objet).
```

Comment:
That sounds a bit odd. Is this an Object or an Array? An array of objects?

Reply:
I might be mixing up JSON terminology with earth systems terminology. But the intent is that the thing in the STAC collection is a JSON object like

```json
"cube:variables": {
    "variable_names: {
        // variable definition here
   }
}
```

Maybe "describes a multi-dimensional array" rather than "defines"?  Or maybe we avoid "array" entirely, since it has multiple meanings, and just use the word "variable" everywhere.

Reply:
I'd probably go with something like:

> A *Variable Object* describes a variable, which is usually a multi-dimensional array. Each variable has dimensions, which are described by [Dimension Objects](#dimension-objet).

And then in the following always use variable... What do you think?